server-spi-private/src/main/java/org/keycloak/events/EventType.java

```diff
@@ -189,6 +189,13 @@ public enum EventType implements EnumWithStableIndex {
 
     USER_SESSION_DELETED(61, false),
     USER_SESSION_DELETED_ERROR(0x10000 + USER_SESSION_DELETED.getStableIndex(), false),
+
+    CREDENTIAL_REQUEST(62, true),
```

Comment:
```suggestion
    VERIFIABLE_CREDENTIAL_REQUEST(62, true),
```

Is it possible to add `VERIFIABLE_` prefix to those events? For example something like `VERIFIABLE_CREDENTIAL_REQUEST` (same for other types). In Keycloak, the "credentials" mostly refer to the regular user credentials (like for example password, OTP or WebAuthn security key), so would be nice to make this  a bit more clear that those event types are about "verifiable credentials" .